src/test/java/org/antonkhmarun/tests/CreatingAccountTest.java

```diff
@@ -0,0 +1,165 @@
+package org.antonkhmarun.tests;
+
+import com.github.javafaker.Faker;
+import org.antonkhmarun.config.ConfProperties;
+import org.antonkhmarun.enums.Country;
+import org.antonkhmarun.enums.State;
+import org.antonkhmarun.pages.Authentication;
+import org.antonkhmarun.pages.PersonalInformation;
+import org.antonkhmarun.support.RandomDateGenerator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.support.ui.Select;
+
+import java.text.ParseException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CreatingAccountTest extends BaseTest{
+
+    public static PersonalInformation personalInformation;
+    public static Authentication authentication;
+    private final Faker faker = new Faker();
+
+    @BeforeEach
+    public void beforeEachTest() {
+        personalInformation = new PersonalInformation(driver);
+        authentication = new Authentication(driver);
+        driver.get(ConfProperties.getProperty("loginPage"));
+    }
+
+    @Test
+    public void createAccountTest() throws ParseException {
+        authentication.inputEmailCreate(faker.internet().emailAddress());
+        authentication.clickCreateAccountBtn();
+
+        personalInformation.clickRandomRadioBtnMrMrs();
+
+        String firstName = faker.name().firstName();
+        String lastName = faker.name().lastName();
+        fillAllFieldsWithRandomValidData(firstName, lastName);
+
+        personalInformation.clickSubmitAccountBtn();
+
+        assertEquals(firstName + " " + lastName, authentication.getAccountName());
+
+        authentication.logout();
```

Comment:
It is also better to move to method with @After annotation
Test shouldn't do anything after assertion